tests/acceptance/features/webUIDeleteFilesFolders/deleteFilesFolders.feature

```diff
@@ -248,7 +248,7 @@ Feature: deleting files and folders
     And as "Alice" file "lorem.txt" should not exist
     And no message should be displayed on the webUI
 
-
+  @skipOnOC10 @skipOnOCIS
```

Comment:
complete scenario should be deleted, because its not a feature anymore

Reply:
Done, thanks